extensions/amp-consent/0.1/test/test-consent-policy-manager.js

```diff
@@ -87,32 +91,38 @@ describes.realWin(
         consentInfo = constructConsentInfo(
           CONSENT_ITEM_STATE.ACCEPTED,
           'test',
-          constructMetadata(CONSENT_STRING_TYPE.TCF_V1)
+          constructMetadata(CONSENT_STRING_TYPE.TCF_V1),
+          {
+            'purpose-foo': PURPOSE_CONSENT_STATE.ACCEPTED,
+            'purpose-bar': PURPOSE_CONSENT_STATE.REJECTED,
+          }
         );
         manager.setLegacyConsentInstanceId('ABC');
       });
 
-      it('Initiate consent value', function* () {
-        yield macroTask();
+      it('Initiate consent value', async () => {
```

Comment:
nit: test should read like a sentence (like you have above) e.g. `it('should initiate consent value', ...`

Reply:
I often even see just `it('initiates consent value', ...`